### DISCUSSION TITLE
Item cannot be re-assigned to another organisation

### DISCUSSION THREAD

**Discussion by nickbe** [dani-garcia/vaultwarden#1748] [Q&A]
I created a vault item in one of my folders.
Then I shared the item with an organisation.

After that I cannot find any possibility to remove this assignment anymore or change it at least. 
So anything done my accident would stay assigned.

**Comment by BlackDex** [comment]
This is how Bitwarden works.
If you share a item with an organization, that org will become owner, there is no way to un-share that item.
You can only clone it again for example, or delete it if you have the correct privileges.

**Reply by nickbe** [reply]
Ok. How do I clone a shared item? As far as I can see I can only clone non-shared items

NOTE: Cloning shared items seems to be only possible through the desktop app. In the WebUI I can't find it. Therefore the confusion.

**Reply by nickbe** [reply]
And another little question: So every shared item from an organisation can appear in a different subfolder for each users having at least readonly rights to the item?
